news-explorer/src/utils/protectedRoute.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { Route, Navigate } from 'react-router-dom';
+import { Route, Navigate, Outlet } from 'react-router-dom';
 

```

Comment:
`protectedRoute.js` is a component. It should be in `UpperCase` and it should be in `components`